testsuite/tests/apicast/parameters/test_proxy_config.py

```diff
@@ -0,0 +1,64 @@
+"""
+Test for issue https://issues.redhat.com/browse/THREESCALE-8485
+"""
+
+import pytest
+from packaging.version import Version  # noqa # pylint: disable=unused-import
+
+from testsuite import rawobj, TESTED_VERSION  # noqa # pylint: disable=unused-import
+from testsuite.capabilities import Capability
+from testsuite.gateways.apicast.selfmanaged import SelfManagedApicast
+
+pytestmark = [
+    pytest.mark.sandbag,
+    pytest.mark.required_capabilities(Capability.STANDARD_GATEWAY, Capability.CUSTOM_ENVIRONMENT),
+    pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-8485"),
+    pytest.mark.skipif("TESTED_VERSION < Version('2.13')")
+]
+
+
+@pytest.fixture(scope="module")
+def gateway_kind():
+    """Gateway class to use for tests"""
+    return SelfManagedApicast
+
+
+@pytest.fixture(scope="module")
+def staging_gateway(staging_gateway):
+    """Sets environment for test"""
+    staging_gateway.environ["APICAST_LOAD_SERVICES_WHEN_NEEDED"] = True
+    staging_gateway.environ["APICAST_CONFIGURATION_LOADER"] = "lazy"
+
+    return staging_gateway
+
+
+def test_proxy_config(service, api_client):
+    """
+    Preparation:
+        - Create service
+        - Create 500+ proxy configs for that service
+        - Delete default mapping rule and create new one
+    Test:
+        - that creation of new mapping rule was correctly promoted.
+    """
+    endpoint = service.proxy.list()['endpoint']
+
+    for i in range(500):
```

Comment:
this should make 501 configurations, however slightly larger number will increase a confidence that the number of configuration is really over 500